cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -169,6 +176,34 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
         });
 
         detailRKE2ClusterPage.waitForPage(undefined, 'registration');
+
+        cy.get('.checkbox-label').contains('Insecure:').click();
+
+        createRKE2ClusterPage.commandFromCustomClusterUI().then(($value) => {
+          const registrationCommand = $value.text();
+
+          cy.log(registrationCommand);
+          cy.exec(`echo ${ Cypress.env('customNodeKey') } | base64 -d > custom_node.key && chmod 600 custom_node.key`).then((result) => {
+            cy.log('Creating the custom_node.key');
+            cy.log(result.stderr);
+            cy.log(result.stdout);
```

Comment:
can these be removed?

Reply:
I could remove them but without them the exec command only appear truncated in the Command Log having these allows to see the output when there's error and or output from running commands. Specially useful on remote execution like this case.

Reply:
ah i see